### DISCUSSION TITLE
phpMyAdmin with PHP 5.6.4

### DISCUSSION THREAD

**Discussion by jirisnet** [leokhoa/laragon#443] [General]
Please help how to run (install) phpMyAdmin with PHP 5.6.4

**Comment by Pen-y-Fan** [comment]
See [phpmyadmin.net > downloads](https://www.phpmyadmin.net/downloads/) 4.9.11

I would recommend using Heidi SQL which is included with Laragon.

**Reply by uthali** [reply]
Heidi SQL couldn't import the database?????

**Reply by ppfoong** [reply]
Export the database as SQL file, and import the SQL file in Heidi.

PHPMyAdmin import is similar, but having more options, including import using ODS file. You can save ODS with Excel, LibreOffice/OpenOffice, etc.
